### DISCUSSION TITLE
Using Pixi3D To Create a Mobile Game on iOS and Android

### DISCUSSION THREAD

**Discussion by lunarraid** [jnsmalm/pixi3d#70] [Show and tell]
Hey thought I'd just show some love for this library. I was able to use it to create Wrestle Deck, featuring 3D card battles in multiple arenas, and is pretty performant on both Android and iOS devices. Kudos to the developer for creating such an awesome project. You can check out a video below if you're curious what the game looks like.

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/EX7X7HLriS0/0.jpg)](https://www.youtube.com/watch?v=EX7X7HLriS0)

You can also download it here:
App Store - https://apps.apple.com/us/app/wrestle-deck/id1591951969
Google Play - https://play.google.com/store/apps/details?id=com.wrestledeck

**Comment by jnsmalm** [comment]
Awesome, I'm really glad you showed me this :) I think you are the first one (expect me of course) to use this in a real project. If you have any feedback about the lib, please let me know so I can improve it.

**Comment by lunarraid** [comment]
I'd be really curious to see any other projects that use this. Do you have anything public to show off? I've been a fan of using PixiJS for years (I wrote a react library on top of it complete with a flexbox layout) and being able to use its rendering systems for 3D has been something I've been excited about. Initially, I was looking forward to their in-house Odie3D engine, but since they were picked up by Playco I doubt that will ever see the light of day, publicly, so pushing this engine forward is the way to go. I'll try to see if I can be more active in working on it as well. I had made some modifications to it, but got so busy working on my game I never took the time to make PRs, and you implemented half of my modifications anyway. Also let me know if you have a tip jar anywhere, this engine has been a life saver for my project.

**Reply by jnsmalm** [reply]
I can show you one of the games I have been working on that's is currently in production that uses this lib. These types of games are typically in 2D only but in this game we used Pixi3D to render the exploding blocks that falls down and bounces off. This is one of the reasons I wanted to create a lib which works good together with PixiJS 2D rendering API.

Some streamers are playing it in this video: https://youtu.be/072zCvXYHyc?t=35

**Reply by lunarraid** [reply]
Awesome, very polished.

**Reply by goldenratio** [reply]
love the demos! Love the work @jnsmalm 
Next week I am gonna showcase pixi3d as a viable solution to our already existing pixi.js based game engine to our team. Fingers crossed!
